docs/sphinx/using/backends/simulators.rst

```diff
@@ -97,6 +97,28 @@ To execute a program on the :code:`nvidia-mgpu` target, use the following comman
 
   This backend requires an NVIDIA GPU, CUDA runtime libraries, as well as an MPI installation. If you do not have these dependencies installed, you may encounter either an error stating `invalid simulator requested` (missing CUDA libraries), or an error along the lines of `failed to launch kernel` (missing MPI installation). See the section :ref:`dependencies-and-compatibility` for more information about how to install dependencies.
 
+The :code:`nvidia-mgpu` backend has additional performance improvements to
+help reduce your simulation runtimes, even on a single GPU. One of the
+performance improvements is to fuse multiple gates together during runtime. By
```

Comment:
Maybe explain in a few words what does "fuse multiple gates together during runtime" mean

Reply:
Updated in 88f2272a0